services/workflows-service/src/collection-flow/collection-flow.module.ts

```diff
@@ -58,9 +58,6 @@ import { SalesforceIntegrationRepository } from '@/salesforce/salesforce-integra
     HookCallbackHandlerService,
     FileService,
     WorkflowEventEmitterService,
-    DocumentChangedWebhookCaller,
-    WorkflowCompletedWebhookCaller,
-    WorkflowStateChangedWebhookCaller,
```

Comment:
This caused the webhooks to be dispatched twice, because it was imported here and only in WorkflowModule.

Reply:
Nice catch